shell/mac/AppDelegate.mm

```diff
@@ -109,6 +109,7 @@ - (void)setupViewController {
                              .majorVersion = 3,
                              .minorVersion = 0},
           .swapchainColorTextureFormat = kColorFramebufferFormat,
+          .depthTextureFormat = igl::TextureFormat::S8_UInt_Z24_UNorm,
```

Comment:
This should not ne changed for all apps. Only apps using NanoVG would want to enable the stencil buffer.

Reply:
This line still stands. We should not enable the stencil buffer unconditionally for all apps.